x-pack/plugins/lens/public/xy_visualization/xy_expression.tsx

```diff
@@ -322,6 +322,10 @@ export function XYChart({
     );
   };
 
+  const shouldEnrichThemeWithValueLabels = filteredLayers.some(({ seriesType }) =>
+    seriesType.includes('bar')
+  );
```

Comment:
This logic looks different from the logic in the configuration because it's based only on chart type, not based on the number of series being shown. Can you unify this logic?